doc/api/n-api.md

```diff
@@ -1701,6 +1696,12 @@ run and the native memory pointed by the earlier persistent reference
 will not be freed. This can be avoided by calling
 `napi_delete_reference` in addition to `napi_reference_unref` when possible.
 
+**Experimental behavior changes:**
+When `NAPI_EXPERIMENTAL` is defined, the references can be created for
+all value types. The new supported value types do not support weak reference
+semantic and the values of these types are released when the
+count becomes 0 and cannot be accessed from the reference any more.
+
```

Comment:
```suggestion
Change History:

  * experimental (`NODE_EXPERIMENTAL` is defined):
  
    References can be created for
    all value types. The new supported value types do not support weak reference
    semantic and the values of these types are released when the
    count becomes 0 and cannot be accessed from the reference any more.
```

Reply:
This allows us to add more bullet points in the future, to correspond to subsequent changes we may make.